client/src/document/index.scss

```diff
@@ -93,6 +93,13 @@ a.new {
     }
   }
 
+  a.self-link {
+    font-weight: bold;
+    // font-style: italic;
+    text-decoration: none;
+    color: black;
+  }
```

Comment:
Other than my more general comment here https://github.com/mdn/yari/pull/3866/files#r640521981, all that needs changing here is:

```css
a.self-link {
  color: $neutral-100;
  font-weight: bold;  
  text-decoration: none;
}
```